### ISSUE TITLE
the pretrained backbone on IMAGENET?

### ISSUE THREAD

**Issue by yu-yuri** [yakhyo/YOLOv1-pt#1]
No such file or directory: 'model_best.pth.tar'
Could I get the pretrained model on imagenet? Thanks!

**Comment by yakhyo** [comment]
Hi @yyrui , yes I trained the backbone with several epochs (around 10-15) on ImageNet then used it for training YOLOv1. You can train it without pre-trained weights also but there is high probability of gradient vanishing. You can set pretrained weights to False on this line [https://github.com/yakhyo/yolov1-pytorch/blob/main/main.py#L62](https://github.com/yakhyo/yolov1-pytorch/blob/main/main.py#L62)
from
```
net = YOLOv1(pretrained_backbone=True).to(device)
```
to
```
net = YOLOv1(pretrained_backbone=False).to(device)
```

**Comment by yu-yuri** [comment]
OK. Thanks.
